test/Service/ContainerAbstractServiceFactoryTest.php

```diff
@@ -54,7 +60,7 @@ public function validContainers()
      */
     public function testCanRetrieveNamedContainers($serviceName, $containerName)
     {
-        $this->assertTrue($this->services->has($serviceName), "Container does not have service by name '$serviceName'");
+        $this->assertTrue($this->services->has($serviceName, true), "Container does not have service by name '$serviceName'");
```

Comment:
The optional second argument to `has()` was dropped in a [recent patch to the SM develop branch](zendframework/zend-servicemanager#49).